package.json

```diff
@@ -5,10 +5,12 @@
     "node": ">=12.13.0"
   },
   "scripts": {
-    "build": "export NODE_OPTIONS=--openssl-legacy-provider; yarn build:prod && yarn build:node",
```

Comment:
Are you confident that we don't need --openssl-legacy-provider? Is there a risk that some users who are using obsolete cryptography will do longer be able to access ExUI?

Reply:
yeah it doesn't seem to work locally with it, I was getting this previously (can be seen on any Jenkins build):
```

[2023-09-29T13:29:51.767Z] + yarn build

[2023-09-29T13:29:52.368Z] command not found: export
```

when I removed export I got an error saying this NODE_OPTIONS value was not allowed